### PR TITLE
Fridiculous/crazy fips

### PR DESCRIPTION
**Pull request by fridiculous**

### CODE REVIEW

File: libs/datasets/sources/fips_population.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ class Fields(object):
 
     def __init__(self, path):
         data = pd.read_csv(path, dtype={"fips": str})
+        data['fips'] = data.fips.str.zfill(5)
```
Comment (ghop02) [inline libs/datasets/sources/fips_population.py:43]: Can you move this into the `standardize_data` function? would prefer to have all of that logic there

Comment (ghop02) [reply libs/datasets/sources/fips_population.py:43]: Also, i'm curious which fips didn't have padded zeros? 
when I run, it shows they're all length 5 (or na for states):
```
pop = FIPSPopulation.local().population().data.fips
pop.groupby(pop.apply(lambda x: x if pd.isna(x) else len(x))).size()
```

Comment (fridiculous) [reply libs/datasets/sources/fips_population.py:43]: this whole needs to be re-done

Comment (ghop02) [reply libs/datasets/sources/fips_population.py:43]: I would rather have a check that explicitly breaks if the fips isn't len 5 I think. but just my 0.02

**Comment by fridiculous** [comment]
> ohh wait i think this should be balcklisted!

is that because it's a dupe key?

File: libs/build_dod_dataset.py [hunk 2]
```diff
@@ -347,6 +352,7 @@ def get_usa_by_states_df():
     states_final = states_final.fillna(NULL_VALUE)
     states_final['Combined Key'] = states_final['Province/State']
     states_final['State/County FIPS Code'] = states_final['Province/State'].map(us_fips)
+    states_final['State/County FIPS Code'] = states_final['State/County FIPS Code'].astype(str).str.zfill(5)
```
Comment (vincentwoo) [inline libs/build_dod_dataset.py:355]: is this necessary if we have performed the padding earlier in this function at raw_df?

Comment (fridiculous) [reply libs/build_dod_dataset.py:355]: yea, it's probably overkill, then again, most of this code is.

**Pull request merged**